pkg/portal/portal.go

```diff
@@ -261,7 +262,8 @@ func (p *portal) unauthenticatedRoutes(r *mux.Router) {
 }
 
 func (p *portal) aadAuthenticatedRoutes(r *mux.Router) {
-	for _, name := range AssetNames() {
+	names, _ := assets.EmbeddedFiles.ReadFile("v1/portalfiles.txt")
```

Comment:
Should we use https://pkg.go.dev/embed#FS.ReadDir instead? It might be a hassle to keep portalfiles updated.

Reply:
@hawkowl - Thanks Amber for reviewing and suggestion, I will check to see if `FS.ReadDir` can read the files from the different sub-folders respectively.